### DISCUSSION TITLE
Updates of dps

### DISCUSSION THREAD

**Discussion by HGRT846** [make-all/tuya-local#1672] [Q&A] [answered]
Hi, I have a problem with power, voltage update of my sockets, as I read [here](https://github.com/make-all/tuya-local/commit/919ea251881622df64c9b6b98b896831b429330f) it should be updating every 30 sec. by default, but it does not, I have to manually add dps to `self._force_dps = [] ` to them to be updated, maybe I just blind and missed something? Thanks in advance

**Comment by aver-ua** [comment]
hi. I have similar problem with my device (metering circuit breaker). 
please read #1483 discussion
it seems that some devices does not send sensor values even with "force_dps"

**Reply by HGRT846** [reply]
I think I found solution...
I'm gonna flash my sockets/IR remotes with tasmota/esphome, f*ck tuya, I never gonna buy that shit anymore... It's not compatible with HA... I have zigbee in my smarthome, and I never had problem with it. Hope you will find a better solution

**Comment by make-all** [comment, marked as answer]
You need to use the `smartplugv2_polled_power` config. The commit you reference above optimistically assumed that there would be no downside, but it turned out a lot of smartplugs that don't need this workaround don't just not need it but cannot tolerate it and start crashing when it is used.   So the change was reverted for the non-device specific configs and a new config made for those devices that require it.

**Reply by HGRT846** [reply]
It worked, thanks, but there are no anymore LED control/Child lock, etc`...
